src/Vehicle/Vehicle.h

```diff
@@ -645,8 +645,11 @@ class Vehicle : public FactGroup
     QString vehicleImageCompass () const;
 
 public slots:
-    void setLatitude(double latitude);
-    void setLongitude(double longitude);
+    /// Sets the firmware plugin instance data associated with this Vehicle. This object will be parented to the Vehicle
+    /// and destroyed when the vehicle goes away.
```

Comment:
That comment was already there. I didn't understand it and I assumed you did understand it :P